### PR TITLE
qa/cephfs: fix minor bug in caps_helper.py's run_mon_cap_tests()

### PR DESCRIPTION
**Pull request by rishabh-d-dave**
When an entity's MON cap contains no FS name in it, the entity has the
access to all the FSs on cluster and not just to the default FS of the
cluster.

Fixes: https://tracker.ceph.com/issues/55558

Related PR: https://github.com/ceph/ceph/pull/32581

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [x] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Comment by rishabh-d-dave** [comment]
@vshankar This can be added to the weekly test run.

**Review by ajarr — CHANGES_REQUESTED** [review]

File: qa/tasks/cephfs/caps_helper.py [hunk 1]
```diff
@@ -13,10 +13,9 @@ def run_mon_cap_tests(self, moncap, keyring):
         fsls = self.run_cluster_cmd(f'fs ls --id {self.client_id} -k '
                                     f'{keyring_path}')
 
-        # we need to check only for default FS when fsname clause is absent
-        # in MON/MDS caps
-        if 'fsname' not in moncap:
-            self.assertIn(self.fs.name, fsls)
+        if 'fsname=' not in moncap:
+            fsls_admin = get_cluster_cmd_op(args=f'fs ls')
```
[the remark this answers is not in the document]
Comment (rishabh-d-dave) [reply qa/tasks/cephfs/caps_helper.py]: Fixed!

File: qa/tasks/cephfs/caps_helper.py [hunk 2]
```diff
@@ -13,10 +13,9 @@ def run_mon_cap_tests(self, moncap, keyring):
         fsls = self.run_cluster_cmd(f'fs ls --id {self.client_id} -k '
                                     f'{keyring_path}')
 
-        # we need to check only for default FS when fsname clause is absent
-        # in MON/MDS caps
-        if 'fsname' not in moncap:
-            self.assertIn(self.fs.name, fsls)
+        if 'fsname=' not in moncap:
+            fsls_admin = self.fs.mon_manager.raw_cluster_cmd(args='fs ls')
```
Comment (ajarr) [inline qa/tasks/cephfs/caps_helper.py:17]: Any reason for not using self.run_cluster_cmd as done in the line above?

Comment (rishabh-d-dave) [reply qa/tasks/cephfs/caps_helper.py]: I was just trying to be closer to the PR to which this commit originally belonged to.

**Comment by rishabh-d-dave** [comment]
jenkins test windows

**Review by ajarr — APPROVED** [review]

**Comment by rishabh-d-dave** [comment]
@vshankar Fixed now - https://pulpito.ceph.com/rishabh-2022-05-27_09:58:18-fs:multifs-wip-vshankar-testing-20220523-112412-distro-basic-smithi/.

**Comment by rishabh-d-dave** [comment]
jenkins test make check arm64

**Comment by vshankar** [comment]
> @vshankar Fixed now - https://pulpito.ceph.com/rishabh-2022-05-27_09:58:18-fs:multifs-wip-vshankar-testing-20220523-112412-distro-basic-smithi/.

Let's get the other issues fixed so that we get a clean test run. I'll merge this PRs post that.

**Review by vshankar — APPROVED** [review]

**Comment by vshankar** [comment]
> > @vshankar Fixed now - https://pulpito.ceph.com/rishabh-2022-05-27_09:58:18-fs:multifs-wip-vshankar-testing-20220523-112412-distro-basic-smithi/.
> 
> Let's get the other issues fixed so that we get a clean test run. I'll merge this PR post that.

As discussed with @rishabh-d-dave - will pick only this PR out of the other 5 PRs in my next run. Rishabh would need some time to figure out the problematic PR from the list:

- https://github.com/ceph/ceph/pull/46165
- https://github.com/ceph/ceph/pull/45036
- https://github.com/ceph/ceph/pull/44774
- https://github.com/ceph/ceph/pull/40434

**Review by vshankar — APPROVED** [review]

**Pull request merged**